src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -1257,6 +1327,8 @@ CascadiaSettings::CascadiaSettings(SettingsLoader&& loader) :
     _activeProfiles = winrt::single_threaded_observable_vector(std::move(activeProfiles));
     _warnings = winrt::single_threaded_vector(std::move(warnings));
     _themesChangeLog = std::move(loader.userSettings.themesChangeLog);
+    _fragmentExtensions = winrt::single_threaded_vector(std::move(loader.fragmentExtensions));

```

Comment:
note that earlier you constructed one, and now you're _overwriting it_.

it may be worth considering not constructing one to begin with.

CascadiaSettings.h R194-195

https://github.com/microsoft/terminal/pull/18559/files#diff-440c829ea1eb9156969710804c029ee66554b4d77b41870aad5dfb06f93a71c1R194